tests/e2e/restart_test.go

```diff
@@ -73,7 +74,8 @@ func (s *RestartTestData) SetUpSuite(c *C) {
 	c.Assert(err, IsNil, Commentf(out))
 
 	// Create a volume on the second datastore
-	out, err = dockercli.CreateVolume(s.config.DockerHosts[1], s.volumeName+"@"+ds2)
+	ds2 = strings.Replace(ds2, "'", "", -1)
+	out, err = dockercli.CreateVolume(s.config.DockerHosts[1], inputparams.CreateFQVN(s.volumeName, ds2))
```

Comment:
API isn't being used anywhere so the command lines should work with the quoted DS names. Isn't that working?